packages/slice-machine/lib/builders/common/Zone/Card/components/ErrorTooltip.tsx

```diff
@@ -7,25 +7,23 @@ interface ErrorTooltip {
   error?: string;
 }
 
-const ErrorTooltip: React.FC<ErrorTooltip> = ({ error }) => {
+export const ErrorTooltip: React.FC<ErrorTooltip> = ({ error }) => {
   if (error) {
     return (
-      <Fragment>
+      <>
         <ReactTooltip type="light" multiline border borderColor={"tomato"} />
         <FaRegQuestionCircle
           color={"tomato"}
           data-tip={error}
           style={{
             position: "relative",
-            top: "1px",
             right: "24px",
-            width: "24px",
+            height: "24px",
```

Comment:
do you know why the width was not working? Maybe we can keep both to be safe?

Reply:
Yeah i wanted to keep it, but it restricted the sizing of the icon for some reason
<img width="232" alt="Screenshot 2022-07-26 at 17 43 49" src="https://user-images.githubusercontent.com/69536010/181063288-adff96e7-c2d1-42d3-9e90-8f20b9c03a05.png">